internal/service/email/email.service.go

```diff
@@ -17,10 +20,14 @@ func (s *serviceImpl) SendEmail(subject string, toName string, toAddress string,
 	to := mail.NewEmail(toName, toAddress)
 	message := mail.NewSingleEmail(from, subject, to, content, content)
 
-	_, err := s.client.Send(message)
+	resp, err := s.client.Send(message)
 	if err != nil {
 		return err
 	}
+	if resp.StatusCode != http.StatusAccepted {
+		return errors.New(fmt.Sprintf("%d status code", resp.StatusCode))
+	}
+
```

Comment:
So, there is a case where there are no errors but it is not successful, right?

Reply:
yes